corehq/tabs/tabclasses.py

```diff
@@ -682,6 +680,14 @@ def sidebar_items(self):
             )
         return items
 
+    @cached_property
```

Comment:
How long is this cached for?

Reply:
[The life of the instance](https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.functional.cached_property), which is created and destroyed during a request.  It's not a redis cache, it's more like `@property` + `@memoized`.